src/elements/SingleReview.jsx

```diff
@@ -42,41 +53,79 @@ export const SingleReview = ({
       });
   };
 
+  if (!reviewLoaded) return "Loading Review ...";
+  if (!review) return <span className="error">404: couldn't find review</span>;
   return (
     <div className="SingleReview">
       <div className="spreadLine">
-        <h2>{title} </h2>
-        <div className="noWrap">
+        <h2>{review.title} </h2>
+        <span className="noWrap">
           {localVotes > 1 ? `${localVotes} votes` : `${localVotes}vote`}{" "}
           <button
             className="voteButton"
             disabled={processingVote}
             onClick={vote}
           >
-            {localVotes > votes ? "▼undo" : "▲vote"}
+            {localVotes > review.votes ? "undo▼" : "vote▲"}
           </button>
-        </div>
+        </span>
       </div>
```

Comment:
I'd consider extracting the voting out into its own component to prevent this singlereview component from bloating up too much as it's very large at the moment